references/classification/README.md

```diff
@@ -144,26 +144,41 @@ torchrun --nproc_per_node=8 train.py\
 Here `$MODEL` is one of `regnet_x_32gf`, `regnet_y_16gf` and `regnet_y_32gf`.
 
 ### Vision Transformer
-
-#### Base models
+#### vit_b_16
 ```
 torchrun --nproc_per_node=8 train.py\
-    --model $MODEL --epochs 300 --batch-size 64 --opt adamw --lr 0.003 --wd 0.3\
+    --model vit_b_16 --epochs 300 --batch-size 64 --opt adamw --lr 0.003 --wd 0.3\
     --lr-scheduler cosineannealinglr --lr-warmup-method linear --lr-warmup-epochs 30\
     --lr-warmup-decay 0.033 --amp --label-smoothing 0.11 --mixup-alpha 0.2 --auto-augment ra\
     --clip-grad-norm 1 --ra-sampler --cutmix-alpha 1.0 --model-ema
 ```
-Here `$MODEL` is one of `vit_b_16` and `vit_b_32`.
 
-#### Large models
+#### vit_b_32
+```
+torchrun --nproc_per_node=8 train.py\
+    --model vit_b_32 --epochs 300 --batch-size 64 --opt adamw --lr 0.003 --wd 0.3\
+    --lr-scheduler cosineannealinglr --lr-warmup-method linear --lr-warmup-epochs 30\
+    --lr-warmup-decay 0.033 --amp --label-smoothing 0.11 --mixup-alpha 0.2 --auto-augment imagenet\
+    --clip-grad-norm 1 --ra-sampler --cutmix-alpha 1.0 --model-ema
+```
+#### vit_l_16
+```
+torchrun --nproc_per_node=8 train.py\
+    --model vit_l_16 --epochs 600 --batch-size 64 --lr 0.5 --lr-scheduler cosineannealinglr\
```

Comment:
I believe that the table at https://github.com/pytorch/vision/pull/5085#issuecomment-1003247327 has the wrong information concerning the number of nodes, the batch-sizes etc. For example the record for `vit_l_16` is definitely wrong as I used 2 nodes instead of 8 that you mention. 

Moreover the commands as is they don't specify how many nodes we used. For some we use 8 nodes, for others 2 which means that if someone tries to reproduce our results it won't work. Typically the commands provided in this file are for 8 gpus (1 node), unless specified otherwise. I recommend to either "translate" the commands to their 8 GPU equivalents (by adjusting the batch-size) or specify the number of nodes per model here.

Reply:
Ok, let me close this PR and include the changes you suggested into #5085 together with the weights.